test/integration/test-preset-brkpt

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+t=${0##*/}; TEST_NAME=${t:5}   # basename $0 with 'test-' stripped off
+
+[ -z "$builddir" ] && builddir=$PWD
+. ${builddir}/check-common.sh
+
+function _preset_brkpt_cleanup {
+  [[ -f "${builddir}/.bashdbrc" ]] && rm -f "${builddir}/.bashdbrc"
+  if [[ -f "${HOME}/.bashdbrc-test-rename" ]]; then
+    mv "${HOME}/.bashdbrc-test-rename" "${HOME}/.bashdbrc"
+  fi
+}
+# setup trap to ensure cleanup of .bashdbrc
+trap _preset_brkpt_cleanup EXIT INT TERM
+
+# rename the ~/.bashdbrc to avoid interference
+if [[ -f "${HOME}/.bashdbrc" ]]; then
```

Comment:
IMHO data outside the repo/build dir should be not modified by tests. I understand that this is there to prevent side-effects. But wouldn't this be necessary for all other tests, too?
But unless we'd like to introduce a new flag like "--no-user-init" I don't see a good fix for this.

Reply:
None of the previous test cases will be interefered by the ~/.bashdbrc since these test cases disable loading the initialization file. The new test case is special since it tests the per-project initialization file alone. The risk of corrupting the per-user configuration is minimal, since the termination, exit, and interrupt are captured. This change isn't likely to affect ordinary user who won't run the bashdb tests.